lib/sparrow/fcm/v1/token_bearer.ex

```diff
@@ -19,15 +19,25 @@ defmodule Sparrow.FCM.V1.TokenBearer do
   end
 
   @spec start_link(Path.t()) :: GenServer.on_start()
-  def start_link(google_json_path) do
-    json = File.read!(google_json_path)
+  def start_link(raw_fcm_config) do
+    json_list =
+      raw_fcm_config
+      |> List.foldl([], fn config, acc ->
+        [
+          config[:path_to_json]
+          |> File.read!()
+          |> Jason.decode!()
+          | acc
+        ]
+      end)
+      |> Jason.encode!()
```

Comment:
why do you decode each config to encode list of them, maybe just put the list as an env?

Reply:
`:goth` expects to get encoded JSON string that is a valid JSON list of objects. I personally have no idea how to convert a list of JSON objects into JSON list of object in a different, clean way. what you're proposing won't work with `goth` unless I'm missing something.

Reply:
That's what I did at first, but there was an encoding error when passing it as a list of encoded JSONs - that's why it is done this way